eternalcombat-plugin/src/main/java/com/eternalcode/combat/CombatSchedulerAdapter.java

```diff
@@ -0,0 +1,30 @@
+package com.eternalcode.combat;
+
+import com.eternalcode.commons.bukkit.scheduler.BukkitSchedulerImpl;
+import com.eternalcode.commons.bukkit.scheduler.MinecraftScheduler;
+import com.eternalcode.commons.folia.scheduler.FoliaSchedulerImpl;
+import java.util.logging.Logger;
+import org.bukkit.plugin.Plugin;
+
+public final class CombatSchedulerAdapter {
+
+    private static final String FOLIA_CLASS = "io.papermc.paper.threadedregions.RegionizedServer";
+
+    private CombatSchedulerAdapter() {
+        throw new IllegalStateException("You can't instantiate this class");
```

Comment:
```suggestion
        throw new UnsupportedOperationException("This is an utility class and cannot be instantiated");
```
nitpick, but it's a standard in our utility classes